### PR TITLE
Fabo/3383 add message about past chain txs

### LINKED ISSUE #3383: Show message about past transactions unavailable in Activity page
**Is your feature request related to a problem? Please describe.**

Cosmos nodes only store the history of the current chain.

**Describe the solution you'd like**

Educate the user about this and mention, that we will bring past txs to the user in the future.
Let's show a message like "Transactions from past chain versions are not displayed here. We are working on showing you your whole transaction history."

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3383

**Description:**

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/wallet/PageTransactions.vue [hunk 1]
```diff
@@ -19,13 +19,24 @@
         />
       </div>
       <br />
+      <TmDataMsg icon="archive">
+        <div slot="title">
+          Past Chain Transactions
```
Comment (jbibla) [inline src/components/wallet/PageTransactions.vue]:
```suggestion
          Looking for older transactions?
```

File: src/components/wallet/PageTransactions.vue [hunk 2]
```diff
@@ -19,13 +19,24 @@
...
+        </div>
+        <div slot="subtitle">
+          Transactions from past chains are not displayed here. If you are
```
Comment (jbibla) [inline src/components/wallet/PageTransactions.vue]:
```suggestion
          Transactions from previous versions of this blockchain aren't currently available.
```

File: src/components/wallet/PageTransactions.vue [hunk 3]
```diff
@@ -19,13 +19,24 @@
...
+          looking forward to accessing those,
```
Comment (jbibla) [inline src/components/wallet/PageTransactions.vue]:
```suggestion
          Let us know if you'd like to access these transactions.
```

File: src/components/wallet/PageTransactions.vue [hunk 4]
```diff
@@ -19,13 +19,24 @@
...
+          <a href="mailto:contact@lunie.io">write us</a>!
```
Comment (jbibla) [inline src/components/wallet/PageTransactions.vue]: this should open intercom, not email anymore

Comment (Bitcoinera) [reply src/components/wallet/PageTransactions.vue]: What if they are on desktop i.e. not mobile? It doesn't work 😆 

Did you test this before changing it? xD

File: src/vuex/modules/intercom.js [hunk 5]
```diff
@@ -14,7 +14,7 @@ export default () => {
     },
     actions: {
       displayMessenger({ state }) {
-        state.intercom.displayMessenger()
+        if (config.mobileApp) state.intercom.displayMessenger()
```
Comment (Bitcoinera) [inline src/vuex/modules/intercom.js]: I don't understand. This seems like it still would only work on mobile

Comment (Bitcoinera) [reply src/vuex/modules/intercom.js]: And I am right



Did this work on your laptop @faboweb ?

Comment (faboweb) [reply src/vuex/modules/intercom.js]: It didn't create an error! "displayMessenger" is only for mobile
For desktop there is a class defined in index.html (line 84)

Comment (Bitcoinera) [reply src/vuex/modules/intercom.js]: Ok, I just saw it.
Quite strangely both messages in PageValidator and PageTransactions are now gone O.o
What happened? Overridden by other PR I guess?

Comment (faboweb) [reply src/vuex/modules/intercom.js]: sadly no, Intercom only works on certain domains.

**Pull request merged**